onnx_coreml/_graph.py

```diff
@@ -3,27 +3,28 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from onnx import numpy_helper
-from typing import Any, Text, Iterable, List, Dict, Sequence, Optional, Tuple
+from onnx import numpy_helper, ValueInfoProto, AttributeProto, GraphProto, NodeProto, TensorProto, TensorShapeProto
+from typing import Any, Text, Iterable, List, Dict, Sequence, Optional, Tuple, Union
 from typing_extensions import Protocol
+import numpy as np
 
 
 class Transformer(Protocol):
     def __call__(self, graph):  # type: (Graph) -> Graph
         pass
 
 
-EdgeInfo = Tuple[Text, Any, Any]
+EdgeInfo = Tuple[Text, TensorProto.DataType, TensorShapeProto]
```

Comment:
This seems to be causing errors on both python 2 and 3: https://travis-ci.org/onnx/onnx-coreml/jobs/349638124 and https://travis-ci.org/onnx/onnx-coreml/jobs/349638125